cdc/model/owner.go

```diff
@@ -100,13 +99,32 @@ func (o *TableOperation) Clone() *TableOperation {
 	return &clone
 }
 
+// TaskWorkload records the workloads of a task
+// the value of the struct is the workload
+type TaskWorkload map[TableID]uint64
```

Comment:
What about this? So we can add more fields if needed.
```
type TaskWorkload map[TableID]struct { something uint64 }
```

Reply:
```
package main

import (
	"fmt"
)

type TaskWorkload map[int]struct{ workload uint64 }

func main() {
	a := make(TaskWorkload)

	// we have to insert value like this ....
	a[1] = struct{ workload uint64 }{workload: 1}
	b := a[1].workload
	fmt.Println(b)
}
```